### PR TITLE
Fixed successful Contact Primary Affiliation Field using Record Type instead of Field API Name

### PR DESCRIPTION
**Pull request by MatthewBlanski**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: src/classes/AffiliationMappingsHealthCheckVMapper.cls [hunk 1]
```diff
@@ -512,7 +512,7 @@ public virtual with sharing class AffiliationMappingsHealthCheckVMapper {
                 Label.stgHCAfflMapContactPrimaryFieldValid, 
                 new List<Object>{
                     affiliationMappingsModel.primaryAffiliationField, 
-                    affiliationMappingsModel.accountRecordTypeFieldValue
+                    accountlookupFieldOnContactFieldModel.name
```
Comment (xinnanlisf) [inline src/classes/AffiliationMappingsHealthCheckVMapper.cls:515]: Very minor one. ‘l’ should be capitalized. Not a big deal atm thou.

**Review by xinnanlisf — APPROVED** [review]
Looks good. Approved.

File: src/classes/AffiliationMappingsHealthCheckVMapper.cls (hunk 1 above)
Comment (MatthewBlanski) [reply src/classes/AffiliationMappingsHealthCheckVMapper.cls]: Done

**Comment by spinkelman** [comment]
QA Passes - Merging

**Pull request merged**